apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -435,6 +437,7 @@ export const PhoneLoginsItems = [
     linkDescription:
       'Vonage is a communication platform as a service (CPaaS) provider for consumers and businesses.',
     isDarkMode: true,
+    hasLightIcon: true,
```

Comment:
This vonage icon still seems to not show correctly in light mode.
Can you double check this one?

https://docs-git-fork-akshat-bhatia73-fix-twitter-logo-supabase.vercel.app/docs/guides/auth

<img width="292" alt="Screenshot 2023-08-25 at 10 26 58" src="https://github.com/supabase/supabase/assets/25671831/4310c7c0-edd2-4d5e-912b-b73d29fdfbdf">

Reply:
Sure, I'll check.

Reply:
I have made the change so please verify if the issue has resolved or not.

Reply:
Awesome, that looks good now 👍